### PR TITLE
Random updates, most notably ISO and INSPIRE constants, validatable big integers, and some bugfixes

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

File: core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java [hunk 1]
```diff
@@ -239,7 +247,7 @@ public static TimeOperator2 from(TimeOperator to) {
         Solid;
 
         public static String asString(GeometryOperand go) {
-            return "gml:" + go.name();
+            return Joiner.on(":").join(GmlConstants.NS_GML_PREFIX, go.name());
```
Comment (autermann) [inline core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java]: I think that may be a little bit too much overhead for concatenating **two** strings....

File: core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        super(numBits, rnd);
+        validate();
+    }
+
+    public ValidatableBigInteger(int bitLength, int certainty, Random rnd) {
+        super(bitLength, certainty, rnd);
+        validate();
+    }
+
+    protected abstract void validate();
```
Comment (autermann) [inline core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java]: That may be good looking, but it's actually [very bad style](http://stackoverflow.com/questions/3404301/whats-wrong-with-overridable-method-calls-in-constructors).

Comment (nuest) [inline core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java]: Ok - what do you think is an alternative?

File: core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java (hunk 1 above)
Comment (nuest) [inline core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java]: So you like ``GmlConstants.NS_GML_PREFIX + ":" + go.name()`` more?

File: core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java (hunk 2 above)
Comment (nuest) [inline core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java]: Maybe using a builder pattern?

Comment (autermann) [inline core/src/main/java/org/n52/iceland/util/ValidatableBigInteger.java]: I would get rid of the abstract class... It's not really adding any functionality and you can't create a `ValidatableBigInteger` without calling a overwritable method or leaking `this`. A `RangedBigInteger` would be possible, but it's not worth the effort as you would have to consider all cases (e.g. `[0,∞)`, `(0,∞), [1,2]`, etc.) Guava's `Range<C>` would come to mind, but you would again leak `this` if you want to use it's `contains()` method...

File: core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java (hunk 1 above)
Comment (autermann) [inline core/src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java]: I think it's way better ;)

**Pull request merged**